dev/django.Dockerfile

```diff
@@ -1,14 +1,30 @@
 FROM python:3.11-slim
+
 # Install system librarires for Python packages:
 # * psycopg2
 RUN apt-get update && \
     apt-get install --no-install-recommends --yes \
     libpq-dev gcc libc6-dev git && \
     rm -rf /var/lib/apt/lists/*
 
+# Set some behaviors for Python.
 ENV PYTHONDONTWRITEBYTECODE 1
 ENV PYTHONUNBUFFERED 1
 
+# Create a normal user (that can be made to match the dev's user stats), falling
+# back to root; if the root user is specified, don't actually run the `adduser`
+# command.
+ARG USERID=0
+ARG LOGIN=root
+ARG GROUPID=0
+ARG GROUPNAME=root
+RUN getent group ${GROUPID} || addgroup --gid ${GROUPID} ${GROUPNAME}
+RUN getent passwd ${USERID} || adduser --allow-bad-names --uid ${USERID} --gid ${GROUPID} --home /home/${LOGIN} $LOGIN
```

Comment:
Now, I get 

```
 => ERROR [django  3/16] RUN getent group 692584961 || addgroup --gid 692584961 domain^users                                                0.4s
------                                                                                                                                           
 > [django  3/16] RUN getent group 692584961 || addgroup --gid 692584961 domain^users:                                                           
0.386 addgroup: To avoid problems, the username should consist only of                                                                           
0.386             letters, digits, underscores, periods, at signs and dashes, and                                                                
0.386             not start with a dash (as defined by IEEE Std 1003.1-2001). For                                                                
0.386             compatibility with Samba machine accounts, $ is also supported                                                                 
0.386             at the end of the username.  (Use the `--allow-all-names' option
0.386             to bypass this restriction.)
```

It seems like we also need to allow "bad" group names. Adding `--allow-all-names` as the error message suggests, seems to fix it.

```suggestion
RUN getent group ${GROUPID} || addgroup --allow-all-names --gid ${GROUPID} ${GROUPNAME}
RUN getent passwd ${USERID} || adduser --allow-bad-names --uid ${USERID} --gid ${GROUPID} --home /home/${LOGIN} $LOGIN
```